bootloaders/eboot/eboot.c

```diff
@@ -121,6 +194,20 @@ int copy_raw(const uint32_t src_addr,
              const uint32_t size,
              const bool verify)
 {
+    const char* __attribute__ ((aligned (4))) fmtt = ":%08x\n\0\0";
+    uint32_t fmt[2];
+    fmt[0] = ((uint32_t*) fmtt)[0];
+    fmt[1] = ((uint32_t*) fmtt)[1];
+
+#if defined (EBOOT_ENABLE_SERIAL_DEBUG)
```

Comment:
Maybe move the ifdef before the variables (otherwise this may generate a warning if DEBUG is off)